### ISSUE TITLE
Doesn't build

### ISSUE THREAD

**Issue by cemkoker** [coreui/coreui#288]
% npm -v
8.1.4

% npm i
npm WARN deprecated urix@0.1.0: Please see https://github.com/lydell/urix#deprecated
npm WARN deprecated resolve-url@0.2.1: https://github.com/lydell/resolve-url#deprecated
npm ERR! code E404
npm ERR! 404 Not Found - GET https://registry.npmjs.org/yallist/-/yallist-4.0.5.tgz - Not found
npm ERR! 404
npm ERR! 404  'yallist@https://registry.npmjs.org/yallist/-/yallist-4.0.5.tgz' is not in this registry.
npm ERR! 404 You should bug the author to publish it (or use the name yourself!)
npm ERR! 404
npm ERR! 404 Note that you can also install from a
npm ERR! 404 tarball, folder, http url, or git url.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/cem/.npm/_logs/2021-11-24T07_59_09_762Z-debug.log

**Comment by colincalnan** [comment]
I got a very similar message, different package, same version

#290

**Comment by colincalnan** [comment]
I think I have found the issue in `package.json`

Search for `"karma-jasmine": "^4.0.5",`
It seems this version is incorrect. The most recent version of this is `4.0.1` - https://www.npmjs.com/package/karma-jasmine

I changed the version in the `package.json` file 

`"karma-jasmine": "^4.0.1",`
Then I ran these commands in the project directory
```
$ rm -rf node_modules
$ npm cache clean --force
$ npm install
```
and it worked!

**Comment by colincalnan** [comment]
@cemkoker , this may not be resolved with the latest version `v4.1.0` please pull the latest changes and checkout the tag to test

```
git pull 
git checkout v4.1.0
```
